calico/reference/etcd-rbac/certificate-generation.md

```diff
@@ -11,7 +11,7 @@ Authority (CA), Certificates and Keys that can be used to authenticate a
 specific user with etcd. There are many different tools that can be used to
 generate these files. This tutorial tries to layout the unique or specific
 details that are needed for each of the different certificates but uses the
-[hack/tls-setup tool from the etcd repo](https://github.com/coreos/etcd/tree/master/hack/tls-setup){:target="_blank"},
+[hack/tls-setup tool from the etcd repo](https://github.com/coreos/etcd/tree/master/calico/tls-setup){:target="_blank"},
```

Comment:
I think this link is still not quite right - it should be the original - the new link is a 404.

Reply:
thanks - fixed! Should be ready for merge now @caseydavenport @fasaxc